docs/sphinx/reference-frontend-api.rst

```diff
@@ -605,7 +605,7 @@ Functions
 .. function:: void obs_frontend_release_tbar(void);
 
    Emulate a mouse button release on the transition bar and determine transition status.
-   
```

Comment:
Unrelated whitespace changes.

Reply:
Will fix. This was automatically added by VSCode as I've set it up to format the documents on save.